nixos/tests/vscodium.nix

```diff
@@ -1,5 +1,6 @@
 let
   tests = {
+    /*# times out after a couple hours
```

Comment:
@Synthetica9 I believe you contributed the wayland part of the test. Can you have a look whether you can fix it?

Reply:
Yeah, turns out it was OCR failing to recognize text properly. Makes me wonder if we can somehow plug into the API that screen readers use instead, would be a lot more reliable methinks

Reply:
#195650